pkg/reconciler/vertex/controller.go

```diff
@@ -105,14 +105,11 @@ func (r *vertexReconciler) reconcile(ctx context.Context, vertex *dfv1.Vertex) (
 		if apierrors.IsNotFound(err) {
 			e := fmt.Errorf("isbsvc %s not found", isbSvcName)
 			vertex.Status.MarkPhaseFailed("ISBSvcNotFound", e.Error())
-			r.recorder.Event(vertex, corev1.EventTypeWarning, "ISBSvcNotFound", e.Error())
 			return ctrl.Result{}, e
 		}
-		r.markPhaseLogEvent(vertex, log, "FindISBSvcFailed", err.Error(), "Failed to get ISB Service", zap.String("isbsvc", isbSvcName), zap.Error(err))
```

Comment:
Is this right? `markPhaseLogEvent` does 3 things, removing it means stopping doing all of them.

Reply:
This isn't right, should revert back to previously just marking the phase and logging the failure.